diff --git vimrc vimrc
index b3282a1..ea4466a 100644
--- vimrc
+++ vimrc
@@ -431,14 +431,14 @@ let g:user_zen_settings = {
 imap {<CR> {}<ESC>i<CR><ESC>O
 
 " NERDTree settings
-nmap wm :NERDTree<cr>
+"nmap wm :NERDTree<cr>
 let NERDTreeIgnore=['\.swp$']
 
-nnoremap <Esc>A <up>
-nnoremap <Esc>B <down>
-nnoremap <Esc>C <right>
-nnoremap <Esc>D <left>
-inoremap <Esc>A <up>
-inoremap <Esc>B <down>
-inoremap <Esc>C <right>
-inoremap <Esc>D <left>
+"nnoremap <Esc>A <up>
+"nnoremap <Esc>B <down>
+"nnoremap <Esc>C <right>
+"nnoremap <Esc>D <left>
+"inoremap <Esc>A <up>
+"inoremap <Esc>B <down>
+"inoremap <Esc>C <right>
+"inoremap <Esc>D <left>
