diff --git syntax_checkers/ruby/mri.vim syntax_checkers/ruby/mri.vim
index 50a8a11..1c9659c 100644
--- syntax_checkers/ruby/mri.vim
+++ syntax_checkers/ruby/mri.vim
@@ -14,7 +14,7 @@ function! SyntaxCheckers_ruby_GetLocList()
     if has('win32') || has('win64')
         let makeprg = 'ruby -W1 -T1 -c '.shellescape(expand('%'))
     else
-        let makeprg = 'RUBYOPT= ruby -W1 -c '.shellescape(expand('%'))
+        let makeprg = 'RUBYOPT= rvm 1.9.3 do ruby -W1 -c '.shellescape(expand('%'))
     endif
     let errorformat =  '%-GSyntax OK,%E%f:%l: syntax error\, %m,%Z%p^,%W%f:%l: warning: %m,%Z%p^,%W%f:%l: %m,%-C%.%#'
 
